### DISCUSSION TITLE
webpack 性能优化 - Alan|前端博客

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#27] [Announcements]
# webpack 性能优化 - Alan|前端博客

webpack学习总结

https://www.alanwang.site/posts/webpack-tutorial/performance-optimization

**Comment by AlanCarte** [comment]
作者您好我想了解您的博客这个点赞的这个功能，可以和您交流一下吗？我想学习一下您的这个点赞的功能

**Reply by 3Alan** [reply]
点赞和评论功能是基于[Giscus](https://giscus.app/zh-CN)实现的，如果你也是基于Docusaurus搭建的博客，可以参考我这篇[文章](https://www.alanwang.site/posts/blog-guides/docusaurus-comment)
